### PR TITLE
Add support for Nvidia Turing extensions

### PR DESCRIPTION
**Pull request by chaoc**

### CODE REVIEW

**Comment by chaoc** [comment]
@s-perron 
Fixed. Thanks for pointing out

File: source/val/validate_builtins.cpp [hunk 1]
```diff
@@ -1556,9 +1558,10 @@ spv_result_t BuiltInsValidator::ValidatePositionAtReference(
         }
         case SpvExecutionModelGeometry:
         case SpvExecutionModelTessellationControl:
-        case SpvExecutionModelTessellationEvaluation: {
+        case SpvExecutionModelTessellationEvaluation: 
+        case SpvExecutionModelMeshNV: {
           // Position can be a per-vertex variable for tessellation control,
-          // tessellation evaluation and geometry shader stages. In such cases
+          // tessellation evaluation, geometry and mesh shader stages. In such cases
```
Comment (dneto0) [inline source/val/validate_builtins.cpp]: I prefer the Oxford comma (a.k.a. serial comma). But I'm just picky. :-)

File: source/val/validate_derivatives.cpp [hunk 2]
```diff
@@ -54,12 +54,33 @@ spv_result_t DerivativesPass(ValidationState_t& _, const Instruction* inst) {
...
+
+      if(_.HasAnyOfExtensions(exts)) {
+          _.function(inst->function()->id())
+              ->RegisterExecutionModelLimitation([opcode](SpvExecutionModel model,
+                  std::string* message) {
+              if (model != SpvExecutionModelFragment &&
+                  model != SpvExecutionModelGLCompute) {
+                  if (message) {
+                      *message =
+                          std::string("Derivative instructions require Fragment execution "
```
Comment (dneto0) [inline source/val/validate_derivatives.cpp]: This message should probably be updated to include the GLCompute case.  But you can fix that later.

**Comment by dneto0** [comment]
I can fix the format.   But it will be in a different PR since I can't update your branch.

**Comment by dneto0** [comment]
Superceded by #1907, which fixes formatting.

**Comment by dneto0** [comment]
Formatting check passed this time.  Great victory!

**Comment by dneto0** [comment]
This is the real one again.

**Comment by dneto0** [comment]
Rebased and pushed into master as 6e2dab2f

**Pull request closed without merging**